tools/star_2.7.11b_personal_genome_generate.cwl

```diff
@@ -0,0 +1,56 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: star_2-7-11a_build_personal_ref
+label: "STAR Personal Reference Genome Index Generator"
+doc: " Allow user to create a custom genome to align aganist"
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/brownm28/star:2.7.11b'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    coresMin: $(inputs.runThreadN)
+    ramMin: ${ return inputs.memory * 1000 }
+
+baseCommand: [mkdir]
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: >-
+      $(inputs.genomeDir)
```

Comment:
I never got the point of using baseCommand like this. Just makes it harder to read
```suggestion
baseCommand: []
arguments:
  - position: 1
    shellQuote: false
    valueFrom: >-
      mkdir $(inputs.genomeDir)
```

Reply:
All your baseCommand are belong to us